test/apis/devfile_recognizer_test.go

```diff
@@ -133,6 +135,100 @@ func TestDetectLaravelDevfile(t *testing.T) {
 	detectDevFiles(t, "laravel", []string{"php-laravel"})
 }
 
+func TestGetUrlWithVersions(t *testing.T) {
```

Comment:
What happens when the min and max version are both the same or when they're both empty

Reply:
If they are both empty we are back in normal so all versions are used and if they have the same version they fetch a specific version.